spec/system/about_page_spec.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+describe "About page", type: :system do
+  fab!(:user)
+  fab!(:group) { Fabricate(:group, users: [user]) }
+  fab!(:image_upload)
+  fab!(:admin) { Fabricate(:admin, last_seen_at: 1.hour.ago) }
+  fab!(:moderator) { Fabricate(:moderator, last_seen_at: 1.hour.ago) }
+
+  before do
+    SiteSetting.title = "title for my forum"
+    SiteSetting.site_description = "short description for my forum"
+    SiteSetting.extended_site_description = <<~TEXT
+      Somewhat lengthy description for my **forum**. [Some link](https://discourse.org). A list:
+        1. One
+        2. Two
+      Last line.
+    TEXT
+    SiteSetting.extended_site_description_cooked =
+      PrettyText.markdown(SiteSetting.extended_site_description)
+    SiteSetting.about_banner_image = image_upload
+    SiteSetting.contact_url = "http://some-contact-url.discourse.org"
+  end
+
+  describe "legacy version" do
+    it "renders successfully for a logged-in user" do
+      sign_in(user)
+
+      visit("/about")
+
+      expect(page).to have_css(".about.admins")
+      expect(page).to have_css(".about.moderators")
+      expect(page).to have_css(".about.stats")
+      expect(page).to have_css(".about.contact")
+    end
+
+    it "renders successfully for an anonymous user" do
+      visit("/about")
+
+      expect(page).to have_css(".about.admins")
+      expect(page).to have_css(".about.moderators")
+      expect(page).to have_css(".about.stats")
+      expect(page).to have_css(".about.contact")
+    end
+  end
+
+  describe "redesigned version" do
+    let(:about_page) { PageObjects::Pages::About.new }
+
+    before do
+      SiteSetting.experimental_redesigned_about_page_groups = group.id.to_s
```

Comment:
To avoid having to make a group for this spec, you could just set this to:

```suggestion
      SiteSetting.experimental_redesigned_about_page_groups = Group::AUTO_GROUPS[:trust_level_1]
```

Reply:
Using an automatic group didn't work because `in_any_groups?` only allows automatic groups if the user is the system user:

https://github.com/discourse/discourse/blob/6fb91b85bac9b3b4500037ea1d69f5e5323d9bef/app/models/user.rb#L545

Reply:
Oh no this isn't the case, sorry forgot to also say that in specs the users don't get their automatic groups built unless you did this on the fabricator:

```ruby
fab!(:current_user) { Fabricate(:user, refresh_auto_groups: true) }
```

If you change the spec to do this it will work.